.npmignore

```diff
@@ -0,0 +1,21 @@
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+
+pids
+logs
+results
+
+npm-debug.log
+node_modules
+coverage
+.jshintrc
+.travis.yml
+.nyc_output/
+test/
```

Comment:
Some people really like their tests in the package, but if the goal here is "small and tight" then they should be excluded IMO.